src/shutdown.cc

```diff
@@ -246,6 +246,24 @@ class subproc_buffer : private cpbuffer<subproc_bufsize>
     }
 };
 
+static bool
+reboot_cmd_unsupported(const shutdown_type_t type)
+{
+    // weed out unsupported values
+    switch (type) {
+#if !defined(RB_HALT_SYSTEM) && !defined(RB_HALT)
+    case shutdown_type_t::HALT: return true;
```

Comment:
match style used elsewhere - case clause should be on a line by itself, put `return true;` on the next line (suitably indented). Same with the other case clauses and default clause.